CallStatusNotifier/WindowsTray.cs

```diff
@@ -0,0 +1,219 @@
+﻿using Models;
+using SkypeCallStatusNotifier;
+using System;
+using System.ComponentModel;
+using System.Drawing;
+using System.Linq;
+using System.Windows.Forms;
+using Particle.SDK;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using Particle.SDK.Models;
+using System.Threading;
+
+namespace CallStatusNotifier
+{
+    internal class WindowsTray
+    {
+        Container component;
+        Icon icon;
+        NotifyIcon trayIcon;
+        readonly ContextMenuStrip rightClickMenu = new ContextMenuStrip();
+
+        readonly ToolStripMenuItem startNotifier = new ToolStripMenuItem();
+        readonly ToolStripMenuItem endNotifier = new ToolStripMenuItem();
+        readonly ToolStripMenuItem triggerBuzzer = new ToolStripMenuItem();
+        readonly ToolStripMenuItem exit = new ToolStripMenuItem();
+        readonly ToolStripSeparator seperator = new ToolStripSeparator();
+        readonly ToolStripSeparator seperatorMain = new ToolStripSeparator();
+        ParticleDevice myDevice = null;
+        bool deviceSuccessfullyInitiated = false;
+
+        public WindowsTray(string[] args)
+        {
+            InitializeComponent();
+            ICallHandler skypeHandler = new SkypeCallHandler();
+            if (args.Length > 0)
+            {
+                Thread.Sleep(90000);
+            }
+            skypeHandler.InitializeEvents(OnCallStarted, OnCallEnded);
+        }
+
+        public void OnCallStarted(string status)
+        {
+            trayIcon.ShowBalloonTip(200
+               , "Call Started"
+               , "Please notify others by clicking on this if you are initiate a client call"
+               , ToolTipIcon.Info
+               );
+        }
+
+        public void OnCallEnded(string status)
+        {
+            trayIcon.ShowBalloonTip(200
+                       , "Call Ended"
+                       , "Call ended, Please notify other"
+                       , ToolTipIcon.Info
+                       );
+            ChackHostNamesAndEndNotifier();
+        }
+
+        private void InitializeComponent()
+        {
+            icon = new System.Drawing.Icon(@"play-normal-1.ico");
```

Comment:
Consider moving this to a resource file
ref: https://msdn.microsoft.com/en-us/library/aa984334(v=vs.71).aspx